pjsip/src/pjsip/sip_transport.c

```diff
@@ -877,6 +877,12 @@ static void transport_send_callback(pjsip_transport *transport,
 
     PJ_UNUSED_ARG(transport);
```

Comment:
So this can be removed now?

Reply:
Yes, but as `transport` is only used for logging with level 5, if `PJ_LOG_MAX_LEVEL` is set to <5, the unused var warning will reappear. So I guess better leave it as is for now?